From 1ba8516045bc8a13fd72e1a85206a04b29d96758 Mon Sep 17 00:00:00 2001
From: Giulio Benetti <giulio.benetti@benettiengineering.com>
Date: Thu, 12 Dec 2019 12:57:29 +0100
Subject: [PATCH] Bug 1603398 - Fix nsinstall.c build failure on RHEL 7

In Linux.mk -DLINUX and -Dlinux must be passed to DEFINES, not
OS_CFLAGS.

Signed-off-by: Giulio Benetti <giulio.benetti@benettiengineering.com>
---
 nss/coreconf/Linux.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nss/coreconf/Linux.mk b/nss/coreconf/Linux.mk
index d07f8a3c5..4b45a6054 100644
--- a/nss/coreconf/Linux.mk
+++ b/nss/coreconf/Linux.mk
@@ -132,7 +132,7 @@ endif
 
 OS_CFLAGS		= $(DSO_CFLAGS) $(OS_REL_CFLAGS) $(ARCHFLAG) -pipe -ffunction-sections -fdata-sections -DHAVE_STRERROR
 ifeq ($(KERNEL),Linux)
-	OS_CFLAGS	+= -DLINUX -Dlinux
+	DEFINES		+= -DLINUX -Dlinux
 endif
 OS_LIBS			= $(OS_PTHREAD) -ldl -lc
 
-- 
2.20.1

